test/openmp-cpp-fortran/downstream/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(openmp-cpp-fortran-user LANGUAGES CXX Fortran)
+
+message(STATUS "${BLT_ROOT_DIR}")
+
+find_package(openmp-cpp-fortran REQUIRED 
+             NO_DEFAULT_PATH
+             PATHS ${base_install_dir}/lib/cmake/openmp-cpp-fortran)
+
+add_executable(openmp-cpp-fortran-user openmp-cpp-fortran-user.F)
+target_link_libraries(openmp-cpp-fortran-user PUBLIC openmp-cpp-fortran)
+
+set_source_files_properties(
+    openmp-cpp-fortran-user.F
+    PROPERTIES
+    Fortran_FORMAT FREE)
+
+get_target_property(imported_omp_flags openmp INTERFACE_COMPILE_OPTIONS)
+# The below print statement should emit the generator expression for the OMP flags,
+# which is necessary because the flag is CPP and Fortran compiler dependent.
+message(STATUS "${imported_omp_flags}")
```

Comment:
Suggestion: Should this be a test that the `imported_omp_flags` are as expected, rather than simply printing the value?

Reply:
Done